modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -501,30 +501,37 @@ class InboundTransfersModel {
             const mojaloopResponse = shared.internalBulkQuotesResponseToMojaloop(response);
 
             // create our ILP packet and condition and tag them on to our internal quote response
+            this._logger.info(JSON.stringify({bulkQuoteRequest}));
```

Comment:
I am not sure its a good idea to "stringify" a bulk message like this?

Reply:
Ah these stringifyies were left overs from debuging. They can be removed.

Reply:
Removed in https://github.com/mojaloop/sdk-scheme-adapter/pull/404